specific_use_cases/DFU_work_in_progress/station_8760s.ipynb

```diff
@@ -0,0 +1,627 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app = ck.Application()\n",
+    "app.select()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7cb2fe31-d427-4fc4-8260-6b2e7b784e81",
+   "metadata": {},
+   "source": [
+    "#### Now retrieve the data, storing it in a variable called 'data'.\n",
+    "The data that will be retrieved is being bias-corrected using historical observations at each station selected. \n",
+    "\n",
+    "The historical observations come from [HadISD](https://catalogue.ceda.ac.uk/uuid/f579035b3c954475922e4b13705a7669). \n",
+    "\n",
+    "The bias-correction procedure happening behind-the-scenes is called [QDM](https://journals.ametsoc.org/view/journals/clim/28/17/jcli-d-14-00754.1.xml), which is designed to preserve trends while allowing for different biases across quantiles. \n",
+    "\n",
+    "The notebook 'localization.ipynb' steps through the bias-correction in more detail if you wish to understand what is happening."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c191bee8-4daa-420a-9ae3-084244deeedb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "data = app.retrieve()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f174c317-54df-424b-9517-aff3b0cef3e0",
+   "metadata": {},
+   "source": [
+    "#### At this point you may wish to load the data \n",
+    "Then you can examine it further, and save time later when you go to export it. \n",
+    "\n",
+    "When you explicitly load the data, only then does it actually perform all of the computations to bias-correct the gridded data to your chosen stations. If you skip this step it will defer this computation until you do some other step that *uses* the data, whether that is to graph it or export it. \n",
+    "\n",
+    "If you have not launched a cluster, the time that this step takes to complete will scale linearly with the number of stations selected."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e938933d-6e84-48a5-b05a-cef46b4f5869",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "data = app.load(data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4fad4df5-c4c7-49fc-a420-318a45d44188",
+   "metadata": {},
+   "source": [
+    "### Examine\n",
+    "A quick preview of the data structure can be seen by executing a cell with the data variable as the final line."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cda83ec9-cadb-4300-8b2d-9062566aa520",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "47d1abf5-1991-4fd9-8344-af5d7925e3ab",
+   "metadata": {},
+   "source": [
+    "We could stop there, but let's do a little more. Now that we have hourly data to work with, we can *improve upon the constructed 8760's that had otherwise been in use!*\n",
```

Comment:
Suggest something like: 
"Now that we have hourly data to work with, we can create and sample from a pool of 8760's which have consistent and realistic time series properties, improving on stochastic sampling methods that had otherwise been in use.